universe/marathon.json.mustache

```diff
@@ -6,52 +6,77 @@
   {{#service.user}}
   "user":"{{service.user}}",
   {{/service.user}}
-  {{#security.secret-name}}
+  {{#service.security.secret-name}}
   "secrets": {
       "serviceCredential": {
-          "source": "{{security.secret-name}}"
+          "source": "{{service.security.secret-name}}"
       }
   },
-  {{/security.secret-name}}
+  {{/service.security.secret-name}}
   "env": {
-      "JENKINS_AGENT_ROLE": "{{roles.jenkins-agent-role}}",
-      "JENKINS_AGENT_USER": "{{advanced.jenkins-agent-user}}",
+      "JENKINS_AGENT_ROLE": "{{service.roles.jenkins-agent-role}}",
+      "JENKINS_AGENT_USER": "{{jenkins-agent.jenkins-agent-user}}",
       "JENKINS_FRAMEWORK_NAME": "{{service.name}}",
       "MARATHON_NAME": "{{service.marathon-name}}",
-      {{#security.secret-name}}
+      {{#service.security.secret-name}}
         "DCOS_SERVICE_ACCOUNT_CREDENTIAL": { "secret": "serviceCredential" },
```

Comment:
This should be `DCOS_SERVICE_ACCOUNT_PRIVATE_KEY`. And the `DCOS_SERVICE_ACCOUNT` option is missing as well. I'm not sure we still need `MESOS_AUTHENTICATEE` and `MESOS_HTTP_AUTHENTICATEE`.